packages/core/android/src/main/kotlin/com/datadog/reactnative/DdSdkImplementation.kt

```diff
@@ -169,6 +172,20 @@ class DdSdkImplementation(
         }
     }
 
+    private fun configureRumAndTracesForLogs(configuration: DdSdkConfiguration) {
+        val rumForLogsEnabled =
+            configuration.additionalConfig?.get(DD_RUM_ENABLE_LOGS) as? Boolean
+                ?: RUM_ENABLE_LOGS_DEFAULT
+        val tracesForLogsEnabled =
+            configuration.additionalConfig?.get(DD_TRACES_ENABLE_LOGS) as? Boolean
+                ?: TRACES_ENABLE_LOGS_DEFAULT
+
+        (datadog as? DatadogSDKWrapper)?.apply {
+            this.setRumForLogsEnabled(rumForLogsEnabled)
+            this.setTracesForLogsEnabled(tracesForLogsEnabled)
```

Comment:
`apply` receiver is already `this`

```suggestion
           setRumForLogsEnabled(rumForLogsEnabled)
           setTracesForLogsEnabled(tracesForLogsEnabled)
```